test/plugins/collections.test.js

```diff
@@ -325,6 +376,15 @@ describe('collection plugin test', () => {
             })
         );
 
+        it('returns 200 when no pipelineIds in payload', () => {
+            delete options.payload.pipelineIds;
+
+            return server.inject(options).then((reply) => {
```

Comment:
I tried doing it but then was pretty confused as to how I could test that functionality since there are no params passed in to update and instead the dirty keys determine what gets updated?